feat(dli): add a new data source to get quotas (#4521)
huaweicloud/terraform-provider-huaweicloud: docs/data-sources/dli_quotas.md

---
subcategory: "Data Lake Insight (DLI)"
---

# huaweicloud_dli_quotas

> Use this data source to get a list of quotas

Reviewer: ```suggestion
Use this data source to get a list of resource quotas for DLI service within HuaweiCloud.
```

Author: done

## Example Usage

```hcl
data "huaweicloud_dli_quotas" "test" {}
```

## Argument Reference

The following arguments are supported:

> * `region` - (Optional, String) Specifies the region in which to query the resource.

Reviewer: ```suggestion
* `region` - (Optional, String) Specifies the region in which to query the resource quotas.
```

Author: This parameter description is automatically generated by PMS.

  If omitted, the provider-level region will be used.

> * `type` - (Optional, String) Specifies the type of quota.

Reviewer: ```suggestion
* `type` - (Optional, String) Specifies the resource type that used to query corresponding quota.
```

Author: done

  Value options:

>    + **CU**: computing unit

Reviewer: ```suggestion
   + **CU**: Computing unit.
```
check the format and do not missing punctuation mark (.).

Author: done
